projects/core/src/lib/util/components/backend-status-page/backend-status-page.component.ts

```diff
@@ -27,4 +28,15 @@ export class BackendStatusPageComponent implements OnInit {
             complete: () => this.changeDetectorRef.markForCheck(),
         });
     }
+
+    refresh(): void {
+        this.error = undefined;
+        this.backendInfo = undefined;
+
+        this.ngOnInit();
```

Comment:
It is strange to call `ngOnInit` yourself, because it is a workflow function in Angular.

Couldn't you reverse it and let `ngOnInit` let call refresh?